data/apps.yml

```diff
@@ -1,119 +1,71 @@
   icons:
-    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.kde.kdenlive.png"
-    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.kde.kdenlive.png"
-    
-- name:     "Skype"
-  id:       "com.skype.Client"
-  ref:      "app/com.skype.Client/x86_64/stable"
-  summary:  "Call and message Skype users"
-  color:    "#C5BFB5"
-  "flatpakref": "https://flathub.org/repo/appstream/com.skype.Client.flatpakref"
-  icons:
-    "64":   "https://lh3.googleusercontent.com/QfAEt_ya6-n8w_TD9-PsghFC2DMSO7fLGNZB4cQ3RtbBbHFkXJE_gxOc3l32-j6LXg=w64"
+    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.libretro.RetroArch.png"
+    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.libretro.RetroArch.png"
 
-- name:     "Google Play Music"
-  id:       "com.googleplaymusicdesktopplayer.GPMDP"
-  ref:      "app/com.googleplaymusicdesktopplayer.GPMDP/x86_64/stable"
-  summary:  "Google Play Music desktop app"
-  color:    "#FAA22B"
-  featured: "yes"
-  "flatpakref": "https://flathub.org/repo/appstream/com.googleplaymusicdesktopplayer.GPMDP.flatpakref"
+- name:     "GNOME Boxes"
+  id:       "org.gnome.Boxes"
+  ref:      "app/org.gnome.Boxes/x86_64/stable"
+  summary:  "Virtualization made simple"
+  "flatpakref": "https://flathub.org/repo/appstream/org.gnome.Boxes.flatpakref"
   icons:
-    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/com.googleplaymusicdesktopplayer.GPMDP.png"
-    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/com.googleplaymusicdesktopplayer.GPMDP.png"
-    
-- name:     "VLC"
-  id:       "org.videolan.VLC"
-  ref:      "app/org.videolan.VLC/x86_64/stable"
-  summary:  "Open-source multimedia player"
-  color:    "#DB946C"
-  "flatpakref": "https://flathub.org/repo/appstream/org.videolan.VLC.flatpakref"
+    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.gnome.Boxes.png"
+    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.gnome.Boxes.png"
+
+- name:     "LibreOffice"
+  id:       "org.libreoffice.LibreOffice"
+  ref:      "app/org.libreoffice.LibreOffice/x86_64/stable"
+  summary:  "The LibreOffice productivity suite"
+  "flatpakref": "https://flathub.org/repo/appstream/org.libreoffice.LibreOffice.flatpakref"
   icons:
-    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.videolan.VLC.png"
-    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.videolan.VLC.png"
+    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.libreoffice.LibreOffice.png"
+    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.libreoffice.LibreOffice.png"
 
-- name:     "Blender"
-  id:       "org.blender.Blender"
-  ref:      "app/org.blender.Blender/x86_64/stable"
-  summary:  "3D modelling and animation"
-  color:    "#D89C7F"
-  "flatpakref": "https://flathub.org/repo/appstream/com.slack.Slack.flatpakref"
+- name:     "Flatseal"
```

Comment:
I think the target audience for flatpak.org is mostly people who are new to Flatpak, and the target audience for Flatseal is advanced users, so maybe not a good fit?

Reply:
This currently won't show because the frontend code limits to 7 applications. I think showcasing Flatseal is still a good idea even if a bit iffy because it risks breaking things.

Worth also noting that none of these applications are clickable so it's likely that someone would only recognize the icon rather than actively click on it to install it.

Reply:
Maybe flathub.org is a better place to showcase Flatseal than flatpak.org